compact/pid/drich.xml

```diff
@@ -15,10 +15,10 @@
 <constant name="DRICH_window_thickness"   value="0.1*cm"/>  <!-- thickness of entrance and exit walls -->
 <constant name="DRICH_num_sectors"        value="6"/>  <!-- number of azimuthal sectors -->
 <!-- snout geometry: cone with front radius rmax0 and back radius of rmax1 -->
-<constant name="DRICH_rmax0"              value="110.0*cm"/>
+<constant name="DRICH_rmax0"              value="90.0*cm"/>
 <constant name="DRICH_snout_length"       value="20.0*cm"/>
 <constant name="DRICH_snout_slope"        value="DRICH_rmax0 / DRICH_zmin"/> <!-- TODO: increase slope to allow more space for aerogel cones ? -->
-<constant name="DRICH_rmax1"              value="DRICH_rmax0 + DRICH_snout_length * DRICH_snout_slope"/>
+<constant name="DRICH_rmax1"              value="DRICH_rmax0 + DRICH_snout_length *(0.20+ DRICH_snout_slope)"/>
```

Comment:
Small suggestion, just to prevent my own future confusion: remove this extra `DRICH_snout_slope` variable, since with the extra 200 mrad it is no longer the real slope. And we can remove the `TODO`:
```xml
<constant name="DRICH_rmax1"              value="DRICH_rmax0 + DRICH_snout_length *(0.20 + DRICH_rmax0 / DRICH_zmin")"/>
```

Reply:
This is a good idea. Have you already implemented?

Reply:
No I haven't, please go ahead

Reply:
For example if we don't take the tangent into account the DRICH rmax for (90 cm, 20 cm) scenario will be 94 cm instead of 105 cm. This is substantially different.